app-doc/abs-guide/abs-guide-10.ebuild

```diff
@@ -1,27 +1,26 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="5"
+EAPI=7
 
 DESCRIPTION="An in-depth exploration of the art of shell scripting"
 HOMEPAGE="https://www.tldp.org/LDP/abs/html"
 
 SRC_URI="http://bash.deta.in/abs-guide-final.tar.bz2
 	pdf? ( http://bash.deta.in/abs-guide.pdf )"
+S="${WORKDIR}"/abs
 
 LICENSE="public-domain"
-IUSE="pdf"
 SLOT="0"
 KEYWORDS="~alpha amd64 hppa ~mips ppc sparc x86"
-
-DEPEND=""
-RDEPEND=""
-
-S="${WORKDIR}/abs"
+IUSE="pdf"
 
 src_unpack() {
 	unpack abs-guide-final.tar.bz2
-	use pdf && cp "${DISTDIR}"/abs-guide.pdf "${S}"
+
+	if use pdf ; then
```

Comment:
Why the space here before `;`?

Reply:
sam style ™️, but I think I’m in the minority here